r/R/dplyr-funcs-datetime.R

```diff
@@ -31,7 +31,7 @@ register_bindings_datetime <- function() {
 
     unit <- make_valid_time_unit(unit, c(valid_time64_units, valid_time32_units))
 
-    Expression$create("strptime", x, options = list(format = format, unit = unit))
+    build_expr("strptime", x, options = list(format = format, unit = unit, error_is_null = TRUE))
   })
```

Comment:
Are you thinking of exposing it as an argument in our `strptime` binding? 

We probably don't need that, since someone could do somethign like `arrow_strptime(..., error_is_null = TRUE)` to get that behavior. If we were to expose this to end users, I would recommend doing it via an option, so that all of the various places that use `strptime` in our bindings would all work the same — though even that I'm a bit doubtful someone will want to control. So I would say let's wait for someone to ask for it before implementing it

Reply:
For a moment I forgot we can always call the arrow function if we need it. So 100% agree with you that we do not need to expose the arg to end users.